content/articles/mobx.md

```diff
@@ -0,0 +1,355 @@
+Title: A MobX introduction and case study
+Date: 2016-10-26
+Short_summary: Getting started with MobX and a real example
+Category: Dev
+Authors: Vincent
+
+*Reading time: ~15 minutes.*
+
+[MobX](https://mobxjs.github.io/mobx/) is a state management library for JavaScript frontend application. This article is an introduction to it by example as well as showing how it works in a real app with TypeScript and is based on a talk I gave at the Osaka Web designers and developers meetup recently.
+<!-- PELICAN_END_SUMMARY -->
+
+## What is MobX and why should I look into it
+
+### MobX?
+Let's start by explaining what is MobX and how it works. It's presented as using Transparent Functional Reactive Programming.
+
+Now, FRP is a very controversial term as everyone seems to have their own definition of what it means so let's forget about that and look at a graph from the MobX docs (click on it to open in a new tab and get full size or open [this link](../images/mobx-flow.png)).
```

Comment:
"definition of what it means" is redundant, just "definition" maybe?

> graph

illustration?